master/usage/migrating-pools.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Migrating Active Pools
+---
+
+## About migrating active pools
+
+When using Calico-IPAM, each workload is assigned an address from the configured pools.
+You may want to change the pool being used on an active cluster for any of the following reasons:
+- You are migrating to a different pool with a larger cidr in order to accomodate more workloads.
+- You are transitioning from one cidr to a completely different cidr. The first cidr may have been chosen accidentally.
+
+### Purpose of this page
+
+Provide guidance on how to migrate off of a live Calico pool.
+
+## Prerequisites
+
+**Orchestrator Support**
+
+While calico supports changing the live address range, not all orchestrators do. 
+Be sure to consult the documentation for whichever installer you are using to ensure it supports changing it. 
+
+For example, in Kubernetes, you will also need to change:
+- kube-apiserver's `--pod-network-cidr`
+- kube-proxy & kube-controller-manager's `--cluster-cidr`
+
+**Application Availability Impact**
+
+This process wil require the recreation of all Calico-networked pods, which will have some impact on availability
+of your applications.
+
+**Consequences of deleting of an active Pool without following this migration procedure**
+
+Removing a pool actively being used will be bad. If the pool is removed before all workloads are moved, those workloads may experience networking difficulties, namely Calico will no longer NAT the traffic (if nat-outgoing is enabled) or properly encapsulate traffic (if using IP-IP mode).
+
+Removing an active pool without following the migration procedure will cause network connectivity disruptions in running 
+workloads. Namely, Calico will no longer NAT the traffic (if nat-outgoing is enabled) or properly encapsulate traffic (if using IP-IP mode).
+
+## Migrating off an active pool
+
+The basic process is as follows:
+
+1. Add a new pool.
+2. Disable the old pool.
+3. Recreate all existing workloads that were assigned an address from the old pool.
+4. Remove the old pool.
+
+### Example: Kubernetes
+
+In this example, we just provisioned a Kubernetes cluster using Kubeadm, and realized we set `--pod-network-cidr=10.0.0.0/16`, but forgot to set the default Calico pool to match,
```

Comment:
Really like the illustration with a concrete (and oh so common example!)  but the sentence is a bit long.  How about splitting it up a bit:

    In this example, we created a cluster with Kubeadm.  We wanted the pods to use IPs in the range 
    10.0.0.0/16 so we set `--pod-network-cidr=10.0.0.0/16` when running `kubeadm`.  However, we 
    forgot to set Calico's default pool to match and `kube-dns` has already been allocated an address 
    from the wrong range: